app/views/container_topology/show.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<html class="no-js">
+<head>
+  <meta charset="utf-8">
+
+  <meta name="viewport" content="width=device-width">
+  <style>
+    /*@font-face {*/
+      /*font-family: 'FontAwesome';*/
+      /*src: url('../assets/fonts/fontawesome-webfont.svg') format('svg');*/
+    /*}*/
+    * {
+      box-sizing: border-box;
+      font-family: sans-serif;
+      font-size: 15px;
+    }
+    body {
+      margin: 0px;
+    }
+    kubernetes-topology-graph {
+      border: 10px solid lightgray;
```

Comment:
@abonas I'd suggest changing the border styling to:
border-top: 1px solid #eee;